manager/src/main/java/com/redhat/service/smartevents/manager/dns/kubernetes/DnsServiceMinikubeImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.redhat.service.smartevents.manager.dns.kubernetes;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import com.redhat.service.smartevents.manager.dns.DnsService;
+
+import io.quarkus.arc.properties.IfBuildProperty;
+
+@ApplicationScoped
+@IfBuildProperty(name = "event-bridge.k8s.orchestrator", stringValue = "minikube")
```

Comment:
@r00ta It would be good to send an email to openbridge-dev to make everyone aware of this pattern for optionally enabling concrete bean implementations depending on our runtime. Or perhaps we finally start to put stuff into our `CONTRIBUTING.md`. In any case, trying to be consistent across the team is the goal and we know that @vaibhavjainwiz was also looking for a pattern to do this at some point.